.travis.yml

```diff
@@ -13,18 +13,18 @@ branches:
 
 env:
  matrix:
-  - DJANGO="django>=1.11,<1.12" DATABASE_URL='mysql://root@127.0.0.1/freeradius_test'
-  - DJANGO="django>=1.11,<1.12" DATABASE_URL='postgres://postgres@127.0.0.1/freeradius_test'
-  - DJANGO="django>=1.11,<1.12" DATABASE_URL='mysql://root@127.0.0.1/freeradius_test' SAMPLE_APP=1
-  - DJANGO="django>=1.11,<1.12" DATABASE_URL='mysql://root@127.0.0.1/freeradius_test' SAMPLE_APP=1
+  - DJANGO="django>=1.11,<2.1" DATABASE_URL='mysql://root@127.0.0.1/freeradius_test'
```

Comment:
does this ensure testing both django 1.11 and 2.0 version?

Reply:
i think yes, i dont see any error

Reply:
@phyxkal this is wrong, see my fix

Reply:
the env matrix should look like the following:

```python
env:
 matrix:
  - DJANGO="django>=1.11,<1.12" DATABASE_URL='mysql://root@127.0.0.1/freeradius_test'
  - DJANGO="django>=1.11,<1.12" DATABASE_URL='postgres://postgres@127.0.0.1/freeradius_test'
  - DJANGO="django>=1.11,<1.12" DATABASE_URL='mysql://root@127.0.0.1/freeradius_test' SAMPLE_APP=1
  - DJANGO="django>=2.0,<2.1" DATABASE_URL='mysql://root@127.0.0.1/freeradius_test'
  - DJANGO="django>=2.0,<2.1" DATABASE_URL='postgres://postgres@127.0.0.1/freeradius_test'
  - DJANGO="django>=2.0,<2.1" DATABASE_URL='mysql://root@127.0.0.1/freeradius_test' SAMPLE_APP=1
```

Reply:
owh, so should be separated right ?, ok